specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/machineLearningServices.json

```diff
@@ -2774,6 +2899,33 @@
         "isLatestOsImageVersion": {
           "type": "boolean",
           "description": "Specifies whether this compute instance is running on the latest operating system image."
+        },
+        "osPatchingStatus": {
+          "readOnly": true,
+          "description": "Metadata about the os patching.",
+          "$ref": "#/definitions/OsPatchingStatus"
+        }
+      }
+    },
+    "OsPatchingStatus": {
+      "type": "object",
+      "description": "Returns metadata about the os patching.",
+      "properties": {
+        "patchStatus": {
+          "type": "string",
+          "description": "The os patching status."
+        },
+        "latestPatchTime": {
+          "type": "string",
+          "description": "Time of the latest os patching."
+        },
+        "rebootPending": {
+          "type": "boolean",
```

Comment:
>boolean [](http://example.com/codeflow?start=19&length=7)

Consider using string enums
[https://github.com/Azure/azure-openapi-validator/blob/main/docs/enum-instead-of-boolean.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/enum-instead-of-boolean.md)

Reply:
It will only have 2 values, other boolean type properties in this file are not enums, I'd prefer to keep it as boolean